ingestion/tests/cli_e2e/test_cli_db_base_common.py

```diff
@@ -68,7 +68,7 @@ def assert_for_table_with_profiler(
             self.assertTrue(len(sample_data.rows) == self.inserted_rows_count())
             self.assertTrue(
                 len(lineage["downstreamEdges"][0]["lineageDetails"]["columnsLineage"])
-                == self.inserted_rows_count()
+                == self.view_column_lineage_count()
```

Comment:
we were comparing the lineage the created view with the number of inserted rows(?)